### PR TITLE
Release 2020/07/13

### PR DESCRIPTION
**Pull request by fisx**
## Release Notes

* If you are self-hosting wire on the public internet, consider [changing your brig server config](https://github.com/wireapp/wire-server/blob/49f414add470f4c5e969814a37bc851e26f6d9a7/docs/reference/user/registration.md#blocking-creation-of-personal-users-new-teams-refrestrictregistration).
* Deploy all services except nginz.
* No migrations, no restrictions on deployment order.

## New Features

* Restrict user creation in on-prem installations (#1161)
* Implement active flag in SCIM for user suspension (#1158)

## Bug Fixes

* Fix setting team feature status in Stern/backoffice (#1146)
* Add missing Swagger models (#1153)
* docs/reference/elastic-search.md: fix typos (#1154)

## Internal changes

* Federation: Implement ID mapping (galley) (#1134)
* Tweak cassandra container settings to get it to work on nixos. (#1155)
* Merge wireapp/subtree-hscim repository under `/libs`, preserving history (#1152)
* Add link to twilio message ID format (#1150)
* Run backoffice locally (#1148)
* Fix services-demo (#1149, #1156)
* Add missing license headers (#1143)
* Test sign up with invalid email (#1141)
* Fix ormolu script (source code pretty-printing) (#1142)

### CODE REVIEW

**Comment by fisx** [comment]
I took out #1157 since it's a bit fresh and not urgent.  Instead of leaving it out of the branch, which would have caused (minor) merge conflicts, I reverted it on top of the release branch.  Once the release is merged back into develop, we need to add another un-revert commit to take it back in.

**Comment by fisx** [comment]
CLAAssistant is confused about John's identity due to data missing on github, and can be ignored.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,35 @@
...
+* Deploy all services except nginz.
+* No migrations, no restrictions on deployment order.
+
+## New Features
+
+* Implement active flag in SCIM for user suspension (#1158)
+
+## Bug Fixes
+
+* Restrict user creation in on-prem installations (#1161)
```
Comment (mheinzel) [inline CHANGELOG.md:15]: I'd argue that this is a feature. What do you think?

Comment (fisx) [reply CHANGELOG.md]: I disagree both with the "please squash the changelog commits" and the categorization of #1161, but fine.  :)

Comment (mheinzel) [reply CHANGELOG.md]: Oh, okay. Well, I'm happy to approve now, but I'd be happy to discuss this. Would be nice to understand your perspective.

**Pull request merged**